apiservers/engine/backends/container.go

```diff
@@ -37,18 +40,19 @@ import (
 )
 
 type Container struct {
+	m sync.Mutex
```

Comment:
That's a very good point. Will move the map into struct (assuming swagger won't end up creating a new Container struct per request)